README.md

```diff
@@ -683,7 +683,12 @@ service.findAndVerify("bad-user").attempt.unsafeRunSync()
 
 ### Extras. Derivation
 
-`io.odin.extras.derivation.render` provides a Magnolia-based derivation of the `Render` type class.
```

Comment:
Those changes will be lost on release. Could you change `docs/README.md` instead?